src/chess/piece/Bishop.java

```diff
@@ -5,7 +5,7 @@
 import java.util.Map;
 
 public class Bishop extends Piece {
-  private static final Map<Color, Character> symbols = new HashMap<Color, Character>(){{
+  private static final Map<Color, Character> symbols = new HashMap<Color, Character>() {{
```

Comment:
It was better to commit this part into the other commit but not this `Delete print` commit
And if you modify something related to code style, you have to modify all related parts (this modification itself is right though)